### ISSUE TITLE
[enhancement] Windows wheels?

### ISSUE THREAD

**Issue by miguelsousa** [adobe-type-tools/pyots#46]
I've noticed that this project doesn't release wheels for Windows. Any chance those could be made? Now that [opentype-sanitizer](https://pypi.org/project/opentype-sanitizer/) has been archived, there are no options available to get the latest OTS on Windows. Thanks!

**Comment by josh-hadley** [comment]
Yeah this would be good to have. I've tried to get it working several times with no luck. Perhaps someone with better cibuildwheel/Windows skills would like to pick this up?
